arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -11,209 +11,242 @@ import PublicPreviewBannerPartial from '../partials/_public-preview-banner-parti
 
-```bash
-❯ cargo stylus --help
-Cargo subcommand for developing Stylus projects
+```shell
+Cargo command for developing Stylus projects
 
 Usage: cargo stylus <COMMAND>
 
 Commands:
-  new         Create a new Rust project
+  new         Create a new Stylus project
+  init        Initializes a Stylus project in the current directory
   export-abi  Export a Solidity ABI
-  check       Check a contract
-  deploy      Deploy a contract
-  replay      Replay a transaction in gdb
-  trace       Trace a transaction
-  c-gen       Generate C code
-  help        Print this message or the help of the given subcommand(s)
+  activate    Activate an already deployed contract [aliases: a]
+  cache       Cache a contract using the Stylus CacheManager for Arbitrum chains
+  check       Check a contract [aliases: c]
+  deploy      Deploy a contract [aliases: d]
+  verify      Verify the deployment of a Stylus contract [aliases: v]
+  cgen        Generate c code bindings for a Stylus contract
+  replay      Replay a transaction in gdb [aliases: r]
+  trace       Trace a transaction [aliases: t]
+  help        Print this message or the help of the given command(s)
 
 Options:
   -h, --help     Print help
   -V, --version  Print version
 ```
 
-### Overview
-
-The cargo stylus command comes with useful commands such as `new`, `check` and `deploy`, and `export-abi` for developing and deploying Stylus programs to Arbitrum chains. Here's a common workflow:
+### Creating a project
 
-Start a new Stylus project with
+You can create a new Stylus project by running:
 
-```bash
+```shell
 cargo stylus new <YOUR_PROJECT_NAME>
 ```
 
-The command above clones a local copy of the [stylus-hello-world](https://github.com/OffchainLabs/stylus-hello-world) starter project, which implements a Rust version of the Solidity `Counter` smart contract example. See the [README](https://github.com/OffchainLabs/stylus-hello-world/blob/main/README.md) of stylus-hello-world for more details. Alternatively, you can use `cargo stylus new --minimal <YOUR_PROJECT_NAME>` to create a more barebones example with a Stylus entrypoint locally, useful for projects that don’t need all the Solidity plumbing. Note that the term "minimal" refers to a contract that handles simple bytes in, bytes out operations. Hence, the `cargo stylus export-abi` command (introduced later) won't work with contracts that use the entrypoint style because they don't enforce the Solidity ABI.
+`cargo stylus new` generates a starter template that implements a Rust version of the [Solidity `Counter` smart contract example](https://github.com/OffchainLabs/counter_contract/blob/master/contracts/Counter.sol). 
 
-Then, develop your Rust program normally and take advantage of all the features the [stylus-sdk](https://github.com/OffchainLabs/stylus-sdk-rs) has to offer.
+Alternatively, you can create a more barebone example with a Stylus entry point locally by running `cargo stylus new --minimal <YOUR_PROJECT_NAME>`. 
```

Comment:
If we are showing how to create a project with `cargo stylus new` and the next steps assume you have created a project like that, I would remove the mention to `--minimal` here. If a dev chooses to go the "alternative" path, they will find themselves unable to follow along.

If needed, we can add a new section after the main flow is complete.